src/components/elementView/elementUtils.tsx

```diff
@@ -174,3 +175,9 @@ export const deleteUser = (
   const data = { id: user.id };
   return axios.post(API_ROOT + "api/users/delete", data, headers);
 };
+export const modifyUser = (
```

Comment:
not sure where to put this comment bc idk where it's rendered - but is it possible to rename the inline button to "Edit permissions"? if not then maybe get rid of the text bc they can't edit the whole user so might be misleading?

Reply:
Yeah I had it as that initially but changed it to users bc "edit permissions" made a scroll bar popup and I didn't want to do the CSS to fix it. Fixed it now tho